wnprc_billing/resources/web/wnprc_billing/form/field/InvestigatorField.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2018 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+/**
+ * This field is used to display WNPRC Investigator.
+ */
+Ext4.define('WNPRC_BILLING.form.field.InvestigatorField', {
+    extend: 'EHR_Billing.form.field.EHRBillingRowObserverEntryField',
+    alias: 'widget.wnprc_billing-investigatorfield',
+
+    initComponent: function() {
+        if (!this.originalConfig || !this.originalConfig.lookup) {
+            Ext4.Msg.alert('Error', 'Lookup not defined for combo box');
+        }
+
+        if (!this.originalConfig.lookup.schemaName) {
+            Ext4.Msg.alert('Error', 'Lookup schema not defined for combo box');
+        }
+
+        if (!this.originalConfig.lookup.queryName) {
+            Ext4.Msg.alert('Error', 'Lookup query not defined for combo box');
+        }
+
+        var ctx = LABKEY.moduleContext.ehr;
+        this.containerPath = ctx['EHRStudyContainer'];
```

Comment:
This is a lot of code to copy/duplicate from EHRBillingRowObserverEntryField.js just so that we can change the container we use out of the moduleContext. I wonder if there might be a simpler way where we set a top level property for the EHRBillingRowObserverEntryField component for containerPathField: 'BillingContainer' and use that property in this initComponent for EHRBillingRowObserverEntryField. Then in this override file, just set that top level property to containerPathField: 'EHRStudyContainer'. Thoughts?

Reply:
duplicate code is now removed.